File: Iniciante/1042 - Sort Simples/JavaScript/1042.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const originalNumbers = input.trim().split(/\s+/).map(Number);
  const orderedNumbers = sortNumbers(originalNumbers);
  printArray(orderedNumbers);
  console.log();
  printArray(originalNumbers);
}

function sortNumbers(originalNumbers) {
  const copy = [...originalNumbers];
  copy.sort((a, b) => a - b);
  return copy;
}

function printArray(array) {
  for (const number of array) {
    console.log(number);
  }
}

main();

File: Iniciante/1042 - Sort Simples/Python/1042.py
def main():
    original_numbers = list(map(int, input().split()))
    ordered_numbers = sorted(original_numbers)
    print_array(ordered_numbers)
    print()
    print_array(original_numbers)


def print_array(array):
    for number in array:
        print(number)


if __name__ == "__main__":
    main()
